reports/report_details.py

```diff
@@ -1541,7 +1542,7 @@ def add_transnational_to_regions(regions):
             'name': '100',
             'title': 'Is this story related to Covid19. By major topic, by medium',
             'desc': 'Is this story related to Covid19. By major topic, by medium',
-            'reports': [],
+            'reports': ['global'],
```

Comment:
Why are we changing this? It's not related to this query.

Reply:
That was also a query I tried working on when I created this PR, that's why I made this change. @kilemensi